manifests/n/NickeManarin/ScreenToGif/2.35.0/NickeManarin.ScreenToGif.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.35.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+UpgradeBehavior: install
+ProductCode: '{E32A6F6C-5240-4749-97D5-4C350D9F94DC}'
```

Comment:
```suggestion
```

Reply:
cc @NickeManarin: Is there any way for your CI to update the ProductCode as well?

Invalid ProductCode will not allow your users to upgrade via WinGet.

Reply:
@ItzLevvie Yes, will do.
I had to create this release manually because the CLI refused to do so (previous releases had only one binary).
Strange that the error didn't appear (or at least I didn't see) when testing.
Thanks.